ext/civicrm_search_ui/Civi/Api4/Report.php

```diff
@@ -176,35 +176,23 @@ public static function getFields($checkPermissions = TRUE) {
   }
 
   public static function getEntityOptions(): array {
```

Comment:
these callbacks were effectively doing this themselves, whereas the PR makes that unnecessary as it is done automatically for *all* such callbacks